srai/embedders/geovex/model.py

```diff
@@ -573,3 +575,20 @@ def configure_optimizers(self) -> list["torch.optim.Optimizer"]:
             lr=self.lr,
         )
         return [opt]
+
+    # override get_config to return the model configuration
+    def get_config(self) -> dict[str, int | float]:
+        """
+        Get the model configuration.
+
+        Returns:
+            Dict[str, int | float]: The model configuration.
+        """
```

Comment:
We are still supporting the Python 3.9 version, so it doesn't accept the pipe syntax yet and tests on this Python version failed. Please change to the `Union` version. It will also require additional import at the top from `typing`.
```suggestion
    def get_config(self) -> dict[str, Union[int, float]]:
        """
        Get the model configuration.

        Returns:
            Dict[str, Union[int, float]]: The model configuration.
        """
```